src/core_modules/capture-core/metaData/helpers/EventProgram/getEventProgramEventAccess.js

```diff
@@ -1,12 +1,25 @@
 // @flow
-import { getEventProgramThrowIfNotFound } from './getEventProgramThrowIfNotFound';
+import log from 'loglevel';
+import { errorCreator } from 'capture-core-utils';
+import { getProgramFromProgramIdThrowIfNotFound } from '../getProgramFromProgramIdThrowIfNotFound';
+import { EventProgram } from '../../Program';
 
 export function getEventProgramEventAccess(
```

Comment:
Hey, 
If you make `getEventProgramEventAccess` support both programs types then I recommend renaming the file and the function to something more suitable like `getProgramEventAccess`. Moreover, I think that the file should be moved outside the `EventProgram` folder into a more generic location. WDYT?
Thanks!